tutorials/deploy-remote-desktop-ubuntu-2204/index.mdx

```diff
@@ -0,0 +1,179 @@
+- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)
+- You have an Instance running on Ubuntu Jammy Jellyfish (22.04 LTS)
+- You have sudo privileges or access to the root user
+
+</Message>
+
+## Preparing the remote machine
+
+1. Connect to your Instance using `ssh`:
+
+  ```
+  ssh root@<your_instance_ip>
+  ```
+
+2. Update the `apt` sources list and upgrade the software already installed on the instance:
+
+  ```
+  apt update && apt upgrade -y
+  ```
+
+3. Install the required packages for the desktop environment using the `apt` package manager:
+
+  ```
+  apt install xorg xfce4 xfce4-goodies dbus-x11 x11-xserver-utils
+  ```
+
+The xRDP server binary is available in the default Ubuntu repositories and it can be installed using `apt`.
+
+4. Run the following command installs the packages `xrdp` and `ufw`, a firewall to protect your Instance from unauthorized access:
+
+  ```
+  apt install xrdp ufw
+  ```
+
+The package manager configures the service to be automatically started upon system reboot.
+
+5. Run the following command to check that the service has been started during the installation:
+
+  ```
+  systemctl status xrdp
+  ```
+
+  A message like the following displays, confirming the status `active (running)`:
+
+  ```
+  ● xrdp.service - xrdp daemon
+       Loaded: loaded (/lib/systemd/system/xrdp.service; enabled; vendor preset: enabled)
+       Active: active (running) since Fri 2022-05-06 14:40:08 UTC; 47s ago
+  ```
+
+  In case the service has not been started automatically, start it with `systemctl`:
+
+  ```
+  systemctl start xrdp
+  ```
+
+6. Open the xRDP configuration file in a text editor of your choice, for example `nano`:
+
+  ```
+  nano /etc/xrdp/xrdp.ini
+  ```
+
+5. Scroll to the bottom of the file and add the following line, then save the file and exit the text editor:
+
+  ```
+  exec startxfce4
+  ```
+
+6. Restart the xrdp service using `systemctl` to activate the new configuration:
+
+  ```
+  systemctl restart xrdp
+  ```
+
+## Configuring the firewall
+
+1. Restrict the access to port `3389` to an IP address or a range of IPs. To allow access from the block `192.168.1.0/24` launch the following command:
+
+  ```
+  ufw allow from 192.168.1.0/24 to any port 3389
+  ```
+
+  To allow connections from any host run the following command:
+
+  ```
+  ufw allow 3389
+  ```
+  <Message type="important">
+
+  Exposing the service to any IP may cause a security issue.
+
+  </Message>
+
+2. Enable the firewall:
+
+  ```
+  ufw enable
+  ```
+
+  <Message type="tip">
+
+  For more information on how to configure the firewall of your Instance, you can [read our tutorial about the configuration of ufw on Ubuntu](/tutorials/installation-uncomplicated-firewall/).
+
+  </Message>
+
+## Configuring a regular user
+
+We recommend you create a regular user to connect to the remote desktop, to avoid security issues.
```

Comment:
```suggestion
We recommend you create a regular user on your Instance, other than the root user, to connect to the remote desktop, to avoid security issues.
```